src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();

```

Comment:
I see that you wanted to create a globally accessible storage service, but there are few things wrong about this approach. In case of dynamically changing internal state I'd recommend to follow the Singleton pattern.
1. Make constructors private - no one should be able to instantiate the singleton.
2. Store the only class instance internally as the private static field (`T instance`). You should store the state as a non-static private field - provide necessary accessors and mutators (e.g. T.addFruit(...). 
3. Create `public static T getInstance()` method that provides he internally stored instance:
```java
public static T getInstance() {
  if(instance == null) {
    instance = new T();
  }
  return instance;
}
```